src/scripting/Elsa.Scripting.JavaScript/Extensions/JavaScriptServiceCollectionExtensions.cs

```diff
@@ -13,5 +16,13 @@ public static IServiceCollection AddJavaScriptExpressionEvaluator(this IServiceC
                 .TryAddProvider<IExpressionEvaluator, JavaScriptExpressionEvaluator>(ServiceLifetime.Scoped)
                 .AddNotificationHandlers(typeof(JavaScriptServiceCollectionExtensions));
         }
+
+        public static IServiceCollection ConfigureJavaScriptOptions(this IServiceCollection services, Action<OptionsBuilder<ScriptOptions>> options)
```

Comment:
What's the difference between this method and the `WithJavaScriptOptions` in `ElsaServiceCollectionExtensions`? The method signature and implementation look the same.

Reply:
Sorry, I should have only included the one in the JavaScript service collection extensions. There's now only `WithJavaScriptOptions` in `Elsa.Scripting.JavaScript`